serenity-model/src/main/java/net/thucydides/core/configuration/SystemPropertiesConfiguration.java

```diff
@@ -71,25 +71,17 @@ public SystemPropertiesConfiguration(EnvironmentVariables environmentVariables)
         filePathParser = new FilePathParser(environmentVariables);
     }
 
-    public Configuration copy() {
-        return withEnvironmentVariables(environmentVariables);
-    }
-
-
-    public Configuration withEnvironmentVariables(EnvironmentVariables environmentVariables) {
-        SystemPropertiesConfiguration copy = new SystemPropertiesConfiguration(environmentVariables.copy());
-        copy.outputDirectory = null; // Reset to be reloaded from the System properties
-        copy.defaultBaseUrl = defaultBaseUrl;
-        return copy;
+    @Override
+    public SystemPropertiesConfiguration withEnvironmentVariables(final EnvironmentVariables environmentVariables) {
+        this.environmentVariables = environmentVariables;
```

Comment:
Update the environment variables on the current instance of the object then return a reference to itself.